packages/samples/vuejs/README.md

```diff
@@ -1,19 +1,157 @@
-# vuejs
+# Vue.js Search Page
 
 ## Project setup
-```
+
```

Comment:
I would rename/rephrase these instructions to be just about "here is how you can run this project locally if you desire".

The production build steps is a bit weird, since it's not meant to be used in production at all.